ajax/libs/crypto-js/3.1.9/x64-core.min.js

```diff
@@ -0,0 +1,2 @@
+!function(t,e){"object"==typeof exports?module.exports=exports=e(require("./core.min")):"function"==typeof define&&define.amd?define(["./core.min"],e):e(t.CryptoJS)}(this,function(t){return function(e){var i=t,n=i.lib,o=n.Base,r=n.WordArray,s=i.x64={};s.Word=o.extend({init:function(t,e){this.high=t,this.low=e}}),s.WordArray=o.extend({init:function(t,i){t=this.words=t||[],i!=e?this.sigBytes=i:this.sigBytes=8*t.length},toX32:function(){for(var t=this.words,e=t.length,i=[],n=0;n<e;n++){var o=t[n];i.push(o.high),i.push(o.low)}return r.create(i,this.sigBytes)},clone:function(){for(var t=o.clone.call(this),e=t.words=this.words.slice(0),i=e.length,n=0;n<i;n++)e[n]=e[n].clone();return t}})}(),t});
+//# sourceMappingURL=ajax/libs/crypto-js/3.1.9/x64-core.min.js.map
```

Comment:
I think this sourceMappingURL should be `//# sourceMappingURL=./x64-core.min.js.map`. 
@PeterDaveHello I remember we met this problem before, right?